src/Symfony/Component/Translation/DataCollectorTranslator.php

```diff
@@ -59,7 +59,7 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
     public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null)
     {
         $trans = $this->translator->transChoice($id, $number, $parameters, $domain, $locale);
-        $this->collectMessage($locale, $domain, $id, $trans);
+        $this->collectMessage($locale, $domain, $id, $trans, $parameters);
```

Comment:
Ok, didn't done it because the Twig extension copy the number to the parameters already ([see here](https://github.com/symfony/TwigBridge/blob/498b03a906dd90861701b3d66cc27f2579ac0457/Extension/TranslationExtension.php#L98)),
but when using the translator directly, this is going to be an issue. I will update my commit soon.